apps/core/views.py

```diff
@@ -79,19 +79,24 @@ def import_group(request):
     return render(request, 'import_group.html', data)
 
 
+def get_xml(jobT):
+    """Takes JobTemplate object and returns its prettified job XML"""
+    jobG = JobGen()
+    xml = jobG.getXML(jobT, reserve=True)
+    try:
+        soup = BeautifulSoup(xml, "xml")
+    except:
+        raise Exception("Bad xml format or something is bad")
```

Comment:
Yes, I agree. This code checks valid xml format and it should show all errors. I fixed it 0864e11d4617d6817e98f8070038fc3b26005aa6